client/src/components/geneExpression/cellSetButtons.js

```diff
@@ -29,15 +29,22 @@ class CellSetButton extends React.Component {
     const { differential, eitherCellSetOneOrTwo } = this.props;
     const cellListName = `celllist${eitherCellSetOneOrTwo}`;
     return (
-      <span style={{ marginRight: 10 }}>
-        <Button type="button" onClick={this.set.bind(this)}>
+      <Tooltip
+        content="Select a group of cells, then click here to store them"
```

Comment:
Not sure "store" means anything.   Store in a file? 

Perhaps:   "Save current selection as group one/two."?